sips/sip-034/sip-034.md

```diff
@@ -0,0 +1,188 @@
+# Preamble
+
+SIP Number: 034
+
+Title: Dimension-Specific Tenure Extend Variants 
+
+Author(s):
+
+- Aaron Blankstein <aaron@stackslabs.com>
+- Jude Nelson <jude@stacks.org>
+
+Status: Activation-in-Progress
+
+Consideration: Technical, Governance
+
+Type: Consensus
+
+Layer: Consensus (hard fork)
+
+Created: 2025-10-15
+
+License: CC0-1.0
+
+Sign-off:
+- Brice Dobry <brice@stackslabs.com>, Acting Chair, Technical CAB
+- Jason Schrader <jason@joinfreehold.com>, Chairperson, Governance CAB
+
+Discussions-To: 
+* SIP pull request: https://github.com/stacksgov/sips/pull/236
+* SIP reference implementation: https://github.com/stacks-network/stacks-core/pull/6609
+
+Sign-offs:
+* Technical CAB: https://github.com/stacksgov/sips/blob/feat/sip034/considerations/minutes/technical-cab/2025-10-17-sip-034.md
+* Governance CAB: https://github.com/stacksgov/sips/blob/main/considerations/minutes/governance-cab/2025-10-17-sip-034.md
+
+# Abstract
+
+This SIP details a rider to SIP-033, including a new variant of the tenure change
+control transaction. This variant allows the tenure change to specify a specific 
+dimension of the block budget to reset.
+
+# Copyright
+
+This SIP is made available under the terms of the CC0 (Creative Commons Zero)
+license, available at https://opensource.org/licenses/CC0-1.0. This SIP’s
+copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+
+The Stacks network controls total runtime and I/O expenditure during Nakamoto tenures
```

Comment:
...Nakamoto tenures, I think it makes sense to reference SIP021 here: https://github.com/stacksgov/sips/blob/main/sips/sip-021/sip-021-nakamoto.md#nakamoto-terminology